extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -115,6 +115,8 @@ export const exampleUsage = localize('exampleUsage', "Example Usage");
 export const enterSystemDbName = localize('enterSystemDbName', "Enter a database name for this system database");
 export const databaseNameRequiredVariableOptional = localize('databaseNameRequiredVariableOptional', "A database name is required. The database variable is optional.");
 export const databaseNameServerNameVariableRequired = localize('databaseNameServerNameVariableRequired', "A database name, server name, and server variable are required. The database variable is optional");
+export const otherServer = localize('otherServer', "OtherServer");
+export const otherSeverVariable = localize('otherServerVariable', "$(OtherServer)");
```

Comment:
SSDT allows it, but then there's an error when publishing if you have a variable name like $(ðĶƢ). Should we check for this or just let publishing fail?

Reply:
I think just let publishing fail; if someone hits it and cares, they can submit a dacfx bug.  This way, we don't have to come back and change logic in db projects if that happens and we fix it.

The main reason I asked was that maybe localizing "OtherServer" would signal to users that we encourage them to use localized variable names which, since they're unicode, would fail during publish.  I think maybe we shouldn't localize these two.

Reply:
makes sense. I'll change them to not be localized